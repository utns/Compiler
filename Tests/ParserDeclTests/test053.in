const
    start = 10;
type
    int = integer;
    point = record
        x, y: float;
    end;
var
    a: array[start..30] of array[1..3] of integer;