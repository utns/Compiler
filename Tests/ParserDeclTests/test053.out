integer                  type 
float                    type 
char                     type 
start               const int              10
int                type alias         integer
point              type alias          record
     x                         var           float
     y                         var           float
a                         var array[10..30] of array[1..3] of integer
